activerecord/test/cases/adapters/postgresql/schema_test.rb

```diff
@@ -487,6 +488,14 @@ def test_rename_index
     assert @connection.index_name_exists?("#{SCHEMA_NAME}.#{TABLE_NAME}", new_name)
   end
 
+  def test_dumping_schemas
+    output = dump_all_table_schema(/./)
```

Comment:
You probably wanted to use `/\./`, because otherwise it will still dump everything and make this test case slower than it needs to be.

Reply:
Ah, correct, never mind.